test/k8sT/Services.go

```diff
@@ -648,6 +648,14 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			It("", func() {
 				testNodePort(kubectl, ni, false, false, false, 0)
 			})
+
+			// Explicitly test netfilter compat mode with kube-proxy.
+			SkipItIf(helpers.DoesNotRunOn54Kernel, "with reverse NAT at host netfilterCompatMode=true", func() {
```

Comment:
Shouldn't this be `DoesNotRunOn54OrLaterKernel`?

Reply:
Thats correct, it should be 54 or later kernels.